packages/product-feed-zbozi/easy-coding-standard.yml

```diff
@@ -0,0 +1,10 @@
+imports:
+    - { resource: '%vendor_dir%/shopsys/coding-standards/easy-coding-standard.yml', ignore_errors: true  }
+
+parameters:
+    skip:
+        ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff:
+            - '*/src/ZboziPluginDataFixture.php'
+            - '*/tests/Unit/ZboziFeedTest.php'
+        Shopsys\CodingStandards\Sniffs\ObjectIsCreatedByFactorySniff:
+          - '*/tests/*'
```

Comment:
In some of the config files, there is a new line missing at the end. This results in diffs with worse readability, but it's not that important to fix those - you can leave it as-is.